tests/PHPUnit/Framework/TestCase/IntegrationTestCase.php

```diff
@@ -86,6 +88,15 @@ public function setUp()
             self::restoreDbTables(self::$tableData);
         }
 
+        $extraDefinitions = $this->provideContainerConfig();
```

Comment:
Is this change expected to be in this pull request (it seems unrelated). Either way, if the code is correct, it's fine to have it here, just double checking